README-zh.md

```diff
@@ -9,14 +9,16 @@
 <!--
 This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad that you want to contribute!
 -->
+
 本仓库包含了所有用于构建 [Kubernetes 网站和文档](https://kubernetes.io/) 的软件资产。
 我们非常高兴您想要参与贡献！
 
 <!--
-# Using this repository
+## Using this repository
 
 You can run the website locally using Hugo (Extended version), or you can run it in a container runtime. We strongly recommend using the container runtime, as it gives deployment consistency with the live website.
 -->
+
```

Comment:
Empty lines like this are unnecessary. They are not required by the markdown syntax, and they do not improve readability of the source.

Reply:
> Empty lines like this are unnecessary. They are not required by the markdown syntax, and they do not improve readability of the source.



> Empty lines like this are unnecessary. They are not required by the markdown syntax, and they do not improve readability of the source.

In markdown syntax, an empty line is a new line. Although these are two headings, they will not be connected together, I think we should abide by this standard